NewForce-Cohort-7/tabloid-cli-denim-devil
# Jfogle

# Description

This pull request adds new features to the application, allowing users add and delete their favorite blogs. 

Implemented the "Add a Blog" functionality.
Prompted the user to enter the new blog's title and URL.
Saved the new blog to the database.
Implemented the "Remove a Blog" functionality.
Presented the user with a list of blogs to choose from for removal.
Removed the selected blog from the database.


 
  Able to merge. These branches can be automatically merged.
@JFogle32
Jfogle
 


## Type of change







Testing Instructions
git add and git commit whatever you're working on on your branch
git checkout main
git fetch --a
git checkout <sb-login-ticket1>
If you don't see the most recent version of my files, git pull origin  -jfogle 




# Checklist:
